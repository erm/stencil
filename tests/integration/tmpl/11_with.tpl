{{ foo }}
{% with foo="bar" %}{{ foo }}{% endwith %}
{{ foo }}
