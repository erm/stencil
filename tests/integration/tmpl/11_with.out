1
bar
1
